scipy-1.0/paper.tex

```diff
@@ -606,9 +606,11 @@ \subsection*{SciPy Matures}
 For example, a three-part minisymposium organized for CSE 2009 was
 featured in SIAM News\cite{siamcse09}.
 
-When Paul edited the CiSE May/June 2007 special issue, Python already had a
-strong enough presence in science and engineering so that editors solicited
-the special issue, but it was still niche enough that the average reader
+When Paul Dubois edited the May/June 2007 special issue of the 
+IEEE publication Computing in Science and Engineering (CiSE), 
```

Comment:
@article{dubois2007guest,
  title={Guest Editor's Introduction: Python--Batteries Included},
  author={Dubois, Paul F},
  journal={Computing in Science \& Engineering},
  volume={9},
  number={3},
  pages={7--9},
  year={2007},
  publisher={IEEE}
}

Reply:
added the in-text citation